### ISSUE TITLE
I'm still getting reports the netLog watcher isn't working?

### ISSUE THREAD

**Issue by hastarin** [hastarin/HAST.Elite.Dangerous.DataAssistant#34]
I haven't been able to reproduce it myself.

**Comment by stringandstickytape** [comment]
One problem I had with watchers is that you might get notified as soon as a file creation instruction is issued by some other app to Windows, but _before_ the file has actually been created.  Judicious pauses fixed the problem for me - and it was _very_ machine dependent, never happened on my machine...

From Form1.cs (cos everything's in Form1.cs in RN...)

```
    private void ScreenshotCreated(object sender, FileSystemEventArgs fileSystemEventArgs)
    {
        Thread.Sleep(1000);

        while (!File.Exists(fileSystemEventArgs.FullPath))
        {
            //MessageBox.Show("File created... but it doesn't exist?!  Hit OK and I'll retry...");
            Thread.Sleep(100);
        }
```

**Comment by hastarin** [comment]
@stringandstickytape Thanks!  I'll integrate a similar feature when I get a chance and see if that helps.

**Comment by hastarin** [comment]
Hopefully fixed in c80eab4a406e3d7b68151930d73d30371c39944a
